install/config.yaml

```diff
@@ -45,7 +45,7 @@ spec:
     spec:
       containers:
       - name: wavefront
-        image: vmware/wavefront-adapter-for-istio:0.1.2
+        image: vmware/wavefront-adapter-for-istio:0.1.3
         imagePullPolicy: Always
```

Comment:
This file needs to be generated via helm.

Reply:
Done.